cli-feed-v4.json

```diff
@@ -7172,6 +7172,198 @@
           "default": "true"
         }
       ]
+    },
+    "4.17.0": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimes": {
+        "dotnet": {
+          "net6": {
+            "displayInfo": {
+              "displayName": ".NET 6",
```

Comment:
I think in release 4.16, this is .NET 6.0 (might want to verify all of the displayName values in this release)

Reply:
@phil-allen-msft  Is there anywhere I can look for what these display names are supposed to be?